test/unit/spec/v1/LoginRouter_spec.js

```diff
@@ -38,6 +39,20 @@ import Bundles from 'util/Bundles';
 import { ConfigError, UnsupportedBrowserError } from 'util/Errors';
 import Logger from 'util/Logger';
 import WidgetUtil from 'util/Util';
+
+jest.mock('cross-fetch', () => {
+  const fetchMock = require('jest-fetch-mock');
+  // Require the original module to not be mocked...
+  const originalFetch = jest.requireActual('cross-fetch');
+  return {
+    __esModule: true,
+    ...originalFetch,
+    default: fetchMock
+  };
+});
+import fetch from 'cross-fetch';
+fetch.dontMock();
```

Comment:
What is all the extra mocking set up for?

Reply:
A fetch request is made to retrieve the language bundle, this mocks that request and [this](https://github.com/okta/okta-signin-widget/pull/3418/files#diff-49644429a8fcf0bc21e368acca57de4459f7899f4d600d18e26d866d3230c670R283) is where it is used

Reply:
Oh I mean why can't we use it like this?
```ts
require('jest-fetch-mock').enableMocks();
fetchMock.dontMock();
```

Reply:
So it looks like cross-fetch's default export is a function, forcing us to have to mock it this way, looking at [this](https://stackoverflow.com/a/69931115) reference. 

I did try the above but received an error regarding the mock.